main.c

```diff
@@ -1823,9 +1969,9 @@ void editorProcessKeypress(int c) {
 			editorDelChar(bufr);
 		}
 		break;
-	case CTRL('l'):
-		editorRecenter(bufr);
-		break;
+	case CTRL('l'): {
+		editorRecenter(win);
+	} break;
```

Comment:
Little curious about this syntax - any reason we can't just:

```suggestion
	case CTRL('l'):
		editorRecenter(win);
		break;
```